packages/data/tests/unit/adapters/facts/elide-test.ts

```diff
@@ -426,6 +427,59 @@ module('Unit | Adapter | facts/elide', function(hooks) {
     assert.deepEqual(result, response, 'fetchAsyncQuery returns the correct response payload');
   });
 
+  test('fetchTableExport - success', async function(assert) {
+    assert.expect(2);
+
+    const adapter: ElideFactsAdapter = this.owner.lookup('adapter:facts/elide');
+
+    let response;
+    Server.post(`${HOST}/graphql`, function({ requestBody }) {
+      const requestObj = JSON.parse(requestBody);
+
+      assert.equal(
+        requestObj.query.replace(/__typename/g, '').replace(/[ \t\r\n]+/g, ''),
+        exportFactsQueryStr.replace(/[ \t\r\n]+/g, ''),
+        'fetchTableExport sent the correct query to fetch a tableExport'
+      );
+
+      response = {
+        tableExport: {
+          edges: [
+            {
+              node: {
+                id: requestObj.variables.ids[0],
+                query: 'foo',
+                queryType: 'GRAPHQL_V1_0',
+                status: 'COMPLETE',
+                result: {
+                  httpStatus: '200',
+                  contentLength: 2,
+                  responseBody: JSON.stringify({
```

Comment:
Is this the right shape for a TableExportResponse?

Looks like it should be this shape:
https://github.com/yahoo/navi/pull/1076/files#diff-4c79f1d6c08140eccb2c41d95c498d9dca6ce3383b90127425ba179c7c0b66c9R141